package.json

```diff
@@ -153,5 +153,8 @@
     "xcode": "3.0.1",
     "xml2js": "0.4.23",
     "yazl": "2.5.1"
+  },
+  "directories": {
```

Comment:
Why is that addition needed?

Reply:
I think that came as a part of the npm init i did. I'll remove it.